rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRAData.java

```diff
@@ -101,6 +113,7 @@ public String toString() {
                 "lraId='" + lraId + '\'' +
                 ", clientId='" + clientId + '\'' +
                 ", status='" + status + '\'' +
+                ", statusReason='" + statusReason + '\'' +
```

Comment:
I'm not sure if this was discussed elsewhere - why the `toString` method does not contain the `String` `reportedParticipants`?